releasenotes/notes/use-tls-1.2-by-default-e2983c21056c3359.yaml

```diff
@@ -0,0 +1,6 @@
+---
+upgrade:
+    The Agent now defaults to TLS 1.2 instead of TLS 1.0. The ``force_tls_12`` configuration parameter has been
+    removed since it's now the default behavior.
+    To continue using TLS 1.0 or 1.1, you must set the ``min_tls_version`` configuration parameter to either
```

Comment:
```suggestion
    To continue using TLS 1.0 or 1.1, you must set the `min_tls_version` configuration parameter to either
```

Reply:
With grep I can see that there are both single and double backticks in the release notes but the proportion of double-backticks is way more important so you're might be right

Reply:
The double-backticks are correct.  I think this is the only place DD uses rST instead of Markdown so it's a common, but incorrect, suggested change.